backend/fms_core/viewsets/sample.py

```diff
@@ -89,6 +89,12 @@ class SampleViewSet(viewsets.ModelViewSet, TemplateActionsMixin, TemplatePrefill
             "template": [SAMPLE_METADATA_TEMPLATE["identity"]],
             "importer": SampleMetadataImporter,
         },
+        {
```

Comment:
I would add the action to the library viewset too in order to have the option in the library or sample as well as to the library prefilling. Change the name and description to be specific on the action for the library (the current one is good since you can do both at the sample level).

Reply:
The dropdown for the Robot columns should be removed in the template.